### DISCUSSION TITLE
default rocket address doesn't match .env.template comment

### DISCUSSION THREAD

**Discussion by dionysius** [dani-garcia/vaultwarden#3078] [Q&A] [answered]
### Subject of the issue
I assumed the comments in [.env.template](https://github.com/dani-garcia/vaultwarden/blob/main/.env.template) mean default values (where clearly interpretable as such). But at least for `ROCKET_ADDRESS` this seems to be incorrect. The file states:
> &#35; ROCKET_ADDRESS=0.0.0.0

The actual listen address:
```
root@vaultwarden:~# netstat -tulpen
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       User       Inode      PID/Program name
tcp        0      0 127.0.0.1:8000          0.0.0.0:*               LISTEN      113        52998745   70470/vaultwarden
```

After setting the option explicitly it listens as expected. Either `.env.template` is wrong or the rocket default listen is wrong.

### Deployment environment

* vaultwarden version: 1.27.0

* Install method: built from source

* Clients used: none

* Reverse proxy and version: none

### Steps to reproduce
- Don't set `ROCKET_ADDRESS`
- Start vaultwarden
- Check listen address

### Expected behaviour
Listening on `0.0.0.0`

### Actual behaviour
Listening on `127.0.0.1`

**Comment by stefan0xC** [comment, marked as answer]
Not sure if this is the case for everything but the file clearly states in the beginning that you have to
https://github.com/dani-garcia/vaultwarden/blob/10dadfca068ed449fcd4a74b70ae2cd83990d3d4/.env.template#L3

<del>That means the values in the file are generally **not** the default values.</del> Update: I've compared the defaults from `src/config.rs` and it seems I was incorrect and they are mostly the default values.

**Reply by BlackDex** [reply]
While most values are representing the defaults. Some provide an example of a possible value.

As @stefan0xC that file is to change the defaults.

**Reply by stefan0xC** [reply]
Thanks for the clarification.
